install_config/master_node_configuration.adoc

```diff
@@ -1684,53 +1672,58 @@ comma-delimited list of every host name or IP address you want server
 certificates to be valid for.
 
 [[launching-servers-using-configuration-files]]
-
 == Launching Servers Using Configuration Files
-Once you have modified the master and/or node configuration files to your
+
+After you have modified the master and node configuration files to your
 specifications, you can use them when launching servers by specifying them as an
-argument. Keep in mind that if you specify a configuration file, none of the
-other command line options you pass are respected.
+argument. If you specify a configuration file, none of the other command line
+options you pass are respected.
 
 [NOTE]
 ====
 To modify a node in your cluster, update the xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration maps] as needed.
-Do not manually edit the `node-config.yaml` file.
+Do not manually edit the *_node-config.yaml_* file.
 ====
 
-To launch an all-in-one server using a master configuration and a node
-configuration file:
-
-[options="nowrap"]
+. Launch a master server using a master configuration file:
++
 ----
-$ openshift start --master-config=/openshift.local.config/master/master-config.yaml --node-config=/openshift.local.config/node-<node_hostname>/node-config.yaml
+$ openshift start master \
+    --config=/openshift.local.config/master/master-config.yaml
 ----
 
-To launch a master server using a master configuration file:
-
-[options="nowrap"]
+. Start the network proxy and SDN plug-ins using a node configuration file and a
+*_node.kubeconfig_* file:
++
 ----
-$ openshift start master --config=/openshift.local.config/master/master-config.yaml
+$ openshift start network \
+    --config=/openshift.local.config/node-<node_hostname>/node-config.yaml \
+    --kubeconfig=/openshift.local.config/node-<node_hostname>/node.kubeconfig
```

Comment:
I'd like to request Networking QE @zhaozhanqi to double confirm for `openshift start network`

Reply:
thanks @xingxingxia  LGTM